infrastructures/docker/docker-auth/pom.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2018 Red Hat, Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-infrastructures-docker-parent</artifactId>
+        <groupId>org.eclipse.che.infrastructure.docker</groupId>
+        <version>6.7.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>docker-auth</artifactId>
+    <packaging>jar</packaging>
+    <name>Infrastructure :: Docker :: Docker Auth</name>
+    <properties>
+        <findbugs.failonerror>false</findbugs.failonerror>
```

Comment:
Can it be removed?

Reply:
It should be! Please, do not use this property unless you have strong arguments why we should use it!

Reply:
This came from a copy/paste of the `docker-client` `pom.xml` file. 
I just removed it.